app/assets/stylesheets/bootstrap_and_customization.css.scss

```diff
@@ -63,8 +63,65 @@ img {
   font-weight: bold;
 }
 
+.navbar .navbar-default {
+  height: 80px;
+}
+
+.bs-docs-container,
+.bs-docs-section {
+  padding-top: 77px;
+}
+
+.video-panel {
+  min-height: 369px;
+}
+
+footer {
+  box-shadow: 0 -2px 4px rgba(194, 194, 194, 0.44);
+  margin-top: 25px;
+  padding-bottom: 10px;
+}
+
+a.btn.btn-default {
+  background-color: rgba(255, 255, 255, 0.3);
+  color: #808080;
+
+  &:hover {
+    color: #ffffff;
+    background-color: #019c91;
+    border-color: #019c91;
+  }
+}
+
+.navbar-nav {
+  float: right;
+  margin: -6px 0px;
+  max-height: 77px;
+
+  a {
+    font-size: 18px;
+    margin: 22px 0;
+  }
+}
+
+.navbar-default {
+  height: 78px;
+}
+
+.small-screen-h2 {
+  font-size: 40px;
+  text-shadow: 1px 1px 7px #000000;
```

Comment:
Color literals like `#000000` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Properties should be ordered font-size, margin-left, margin-right, max-width, text-shadow, width